src/shared/components/ErrorBoundary.tsx

```
@@ -0,0 +1,53 @@
+  };
+
+  render() {
+    if (!this.state.hasError) {
+      return this.props.children;
+    }
+
+    const { fallbackRender, fallback } = this.props;
+    if (fallbackRender) {
+      return fallbackRender({ error: this.state.error!, reset: this.reset });
+    }
+    return fallback ?? null;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)